var/spack/repos/builtin/packages/likwid/package.py

```diff
@@ -38,7 +38,8 @@ class Likwid(Package):
     # The reason is that the internal hwloc is patched to contain extra
     # functionality and functions are prefixed with "likwid_".
 
-    depends_on('lua', when='@4.2.0:')
+    depends_on('lua', when='@4.2.0:4.9.9')
+    depends_on('lua@5.2.0', when='@5.0.0:')
```

Comment:
```suggestion
    depends_on('lua', when='@:4')
    depends_on('lua@5.2:', when='@5:')
```

Reply:
Looks good for the moment. I'll fix Lua 5.1 support soon. I'll submit a PR here then.

Reply:
Thanks for cleaning this up.